examples/graphql-interceptors/graphql_interceptors.bal

```diff
@@ -0,0 +1,58 @@
+import ballerina/graphql;
+import ballerina/log;
+
+// Define an interceptor service class named `LogInterceptor`. It cannot have
+// any `resource/remote` methods except the `execute()` remote method.
+readonly service class LogInterceptor {
+
+    // Infers the `graphql:Interceptors` service object from the GraphQL
+    // package.
+    *graphql:Interceptor;
+
+    // Implement the `execute()` remote function provided by the
+    // `graphql:Interceptor` object. Within the function, the `graphql:Context`
+    // and the `graphql:Field` object can be accessed to get the request and
+    // field related information.
+    isolated remote function execute(graphql:Context context,
+                                     graphql:Field 'field)
+                                     returns anydata|error {
```

Comment:
Shall we check the character limits and the correct formatting here?

Reply:
Since the `master` branch uses the earlier format, the character limit is 80.